frontend/js/src/user/components/AddListenModal.tsx

```diff
@@ -65,15 +68,21 @@ export const getListenFromTrack = (
       additional_info: {
         recording_mbid: track.recording.id,
         submission_client: "listenbrainz web",
-        duration_ms: track.length,
         track_mbid: track.id,
-        tracknumber: parseInt(track.number, 10),
+        tracknumber: track.number,
```

Comment:
That was the issue with the `null` values I was describing in the comments on Jira.
I guess I was testing with alphanumeric track numbers for which `parseInt` returns `NaN`.

Reply:
I figured that's what it was, but wasn't sure.

I ended up treating tracknumbers that look like actual numbers differently for display (I wanted to pad the numeric track numbers) in 59ac4dc43